src/main/java/uk/gov/companieshouse/accounts/filing/controller/AccountsFilingController.java

```diff
@@ -1,17 +1,49 @@
 package uk.gov.companieshouse.accounts.filing.controller;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.*;
+
+import uk.gov.companieshouse.accounts.filing.model.FilingRecord;
+import uk.gov.companieshouse.accounts.filing.model.FilingRequest;
+import uk.gov.companieshouse.accounts.filing.model.FilingResponse;
+import uk.gov.companieshouse.accounts.filing.repository.AccountsFilingRepository;
+import uk.gov.companieshouse.logging.Logger;
 
 @Controller
 @RequestMapping("/accounts-filing")
 public class AccountsFilingController {
 
+    private Logger logger;
+    private AccountsFilingRepository accountsFilingRepository;
+
+    @Autowired
+    public AccountsFilingController(Logger logger,AccountsFilingRepository accountsFilingRepository) {
+        this.logger=logger;
+        this.accountsFilingRepository=accountsFilingRepository;
```

Comment:
The spacing here is a bit off should be: 
```suggestion
    public AccountsFilingController(Logger logger, AccountsFilingRepository accountsFilingRepository) {
        this.logger = logger;
        this.accountsFilingRepository = accountsFilingRepository;
```

Reply:
modified